Sources/HipMobileUI/Pages/CharacterSelectionPage.xaml

```diff
@@ -25,7 +25,7 @@
         <views:NotificationExtensionView>
             <views:NotificationExtensionView.ContentTemplate>
                 <Grid x:Name="MainGrid" RowSpacing="0" ColumnSpacing="0">
-                    <Grid x:Name="AdventurerGrid" BackgroundColor="Orange">
+                    <Grid x:Name="AdventurerGrid" BackgroundColor="#FFE57F">
```

Comment:
This should also be referenced from the StaticResources but since this page will be reworked by Sayanti, you don't have to change it.